packages/web3-utils/src/json_rpc.ts

```diff
@@ -28,9 +28,71 @@ import {
 	JsonRpcBatchResponse,
 	JsonRpcSubscriptionResult,
 } from 'web3-types';
-
+import {
+	ParseError,
+	InvalidRequestError,
+	MethodNotFoundError,
+	InvalidParamsError,
+	InternalError,
+	InvalidInputError,
+	ResourcesNotFoundError,
+	ResourceUnavailableError,
+	TransactionRejectedError,
+	MethodNotSupported,
+	LimitExceededError,
+	VersionNotSupportedError,
+	RpcError,
+	rpcErrorValues,
+} from 'web3-errors';
 import { uuidV4 } from './uuid';
 
+export const getRpcError = (code: number) => {
+	const codeStr = code.toString();
+	if (codeStr === '-32700') {
+		return ParseError;
+	}
+	if (codeStr === '-32600') {
```

Comment:
There shouldn't be multiple copies of these error codes in web3js lib multiple places.  we should use error_codes.ts.